js/app/modules/contentGroup/contentGroup.js

```diff
@@ -170,6 +171,7 @@ const ContentGroup = new Module.Class({
             // When the spinner is not being shown on screen, set it to
             // be inactive to help with performance.
             spinner.active = spinner.visible = this._selection.loading;
+            this._error_page.visible = !!this._selection.error;
```

Comment:
This is a great solution and much better than what I was doing previously, which was setting the fonts in the error page to be really small in cases where vertical space made a difference, like in the top menu module. Now that you've improved upon that, consider removing my hacks :smile: